src/libponyrt/mem/alloc.c

```diff
@@ -29,7 +29,10 @@ void* ponyint_virt_alloc(size_t bytes)
 #elif defined(PLATFORM_IS_MACOSX)
   p = mmap(0, bytes, PROT_READ | PROT_WRITE,
     MAP_PRIVATE | MAP_ANON | VM_FLAGS_SUPERPAGE_SIZE_ANY, -1, 0);
-#elif defined(PLATFORM_IS_FREEBSD)
```

Comment:
another interesting difference between dragonfly and freebsd.

note u made the elif a PLATFORM_BSD under the assumption that NetBSD and OpenBSD are like FreeBSD. That could be wrong. Also reversing order would be bad here.

I don't have a better solution.